generate-styles.js

```diff
@@ -25,4 +25,4 @@ names.forEach(function (item) {
 });
 
 console.log(output.slice(0, -2));
-console.log("\{background-color: Crimson;\}");
\ No newline at end of file
+console.log("\{ padding: 5px; background-color: Crimson; color: black; \}");
```

Comment:
`!important` will be needed to actually make the text colour black

Reply:
oh that's stupid, let me test it now